client/my-sites/stats/post-detail-table-section/index.tsx

```diff
@@ -24,33 +25,58 @@ export default function PostDetailTableSection( {
 	const tableOptions = [
 		{ value: 'years', label: translate( 'Months and years' ) },
 		{ value: 'averages', label: translate( 'Average per day' ) },
-		{ value: 'weeks', label: translate( 'Recent Weeks' ) },
+		{ value: 'weeks', label: translate( 'Recent weeks' ) },
```

Comment:
😬 I know this is a necessary change, but let's avoid making mistakes like this in the future.